language/de-DE/joomla.ini

```
@@ -129,42 +130,47 @@ JBROWSERTARGET_NEW="In neuem Fenster öffnen"
+JERROR_LAYOUT_REQUESTED_RESOURCE_WAS_NOT_FOUND="Die angefragte Quelle wurde nicht gefunden."
 JERROR_LAYOUT_SEARCH="Die Website kann durchsucht oder es kann die Startseite aufgerufen werden."
 JERROR_LAYOUT_SEARCH_ENGINE_OUT_OF_DATE_LISTING="Sie über eine Suchmaschine einen <strong>veralteten Index dieser Website</strong> aufgerufen haben."
 JERROR_LAYOUT_SEARCH_PAGE="Seite durchsuchen"
 JERROR_LAYOUT_YOU_HAVE_NO_ACCESS_TO_THIS_PAGE="Sie <strong>keinen Zugriff</strong> auf diese Seite haben."
 JERROR_LOADING_MENUS="Fehler beim Laden der Menüs: %s"
 JERROR_LOGIN_DENIED="Es gibt keinen Zugriff auf die privaten Seiten."
-JERROR_NOLOGIN_BLOCKED="Zugriff verweigert! Entweder ist das Benutzerkonto noch nicht aktiviert oder es wurde gesperrt!"
+JERROR_NOLOGIN_BLOCKED="Zugriff verweigert. Entweder ist das Benutzerkonto noch nicht aktiviert oder es wurde gesperrt."
 JERROR_PAGE_NOT_FOUND="Seite nicht gefunden"
 JERROR_SENDING_EMAIL="E-Mail konnte nicht gesendet werden."
 JERROR_SESSION_STARTUP="Fehler beim Initialisieren der Sitzung."
 JERROR_TABLE_BIND_FAILED="hmm %s ..."
-JERROR_USERS_PROFILE_NOT_FOUND="Das Benutzerprofil konnte nicht gefunden werden!"
+JERROR_USERS_PROFILE_NOT_FOUND="Das Benutzerprofil konnte nicht gefunden werden."
 
 JFIELD_ACCESS_DESC="Zugriffsebene für diese Seite"
 JFIELD_ACCESS_LABEL="Zugriff"
-JFIELD_ALIAS_DESC="Dieser Alias wird in der SEF-URL benutzt. Wird dieser leer gelassen, benutzt Joomla! die Standardwerte dieser Seite. Die Werte sind abhängig von den SEO-Einstellungen (Konfiguration -> Website).<br />Wird Unicode benutzt, werden UTF-8-Aliase ausgegeben. Es ist außerdem möglich, manuell jegliche UTF-8-Buchstaben zu benutzen. Leerzeichen und verbotene Zeichen werden in Bindestriche umgewandelt.<br />Bei der Benutzung von Standard-Umschreibungen, wird der Alias in Kleinbuchstaben ausgegeben und Leerzeichen werden in Striche umgewandelt. Bitte den Alias manuell eintragen. Dabei bitte Kleinbuchstaben und Bindestriche (-) verwenden. Leerzeichen und Unterstriche sind nicht erlaubt. Befinden sich unerlaubte Zeichen im Eintrag, wird als Standardwert Datum und Zeit verwendet."
+JFIELD_ALIAS_DESC="Dieser Alias wird in der SEF-URL benutzt. Wird dieser leer gelassen, benutzt Joomla. die Standardwerte dieser Seite. Die Werte sind abhängig von den SEO-Einstellungen (Konfiguration -> Website).<br />Wird Unicode benutzt, werden UTF-8-Aliase ausgegeben. Es ist außerdem möglich, manuell jegliche UTF-8-Buchstaben zu benutzen. Leerzeichen und verbotene Zeichen werden in Bindestriche umgewandelt.<br />Bei der Benutzung von Standard-Umschreibungen, wird der Alias in Kleinbuchstaben ausgegeben und Leerzeichen werden in Striche umgewandelt. Bitte den Alias manuell eintragen. Dabei bitte Kleinbuchstaben und Bindestriche (-) verwenden. Leerzeichen und Unterstriche sind nicht erlaubt. Befinden sich unerlaubte Zeichen im Eintrag, wird als Standardwert Datum und Zeit verwendet."
```

```suggestion
JFIELD_ALIAS_DESC="Dieser Alias wird in der SEF-URL benutzt. Wird dieser leer gelassen, benutzt Joomla die Standardwerte dieser Seite. Die Werte sind abhängig von den SEO-Einstellungen (Konfiguration -> Website).<br />Wird Unicode benutzt, werden UTF-8-Aliase ausgegeben. Es ist außerdem möglich, manuell jegliche UTF-8-Buchstaben zu benutzen. Leerzeichen und verbotene Zeichen werden in Bindestriche umgewandelt.<br />Bei der Benutzung von Standard-Umschreibungen, wird der Alias in Kleinbuchstaben ausgegeben und Leerzeichen werden in Striche umgewandelt. Bitte den Alias manuell eintragen. Dabei bitte Kleinbuchstaben und Bindestriche (-) verwenden. Leerzeichen und Unterstriche sind nicht erlaubt. Befinden sich unerlaubte Zeichen im Eintrag, wird als Standardwert Datum und Zeit verwendet."
```
Hier ist ein Punkt nicht notwendig.